frontends/web/src/components/transactions/transaction.tsx

```diff
@@ -127,35 +173,41 @@ class Transaction extends Component<Props, State> {
                         <div className={parentStyle.type}>{arrow}</div>
                         <div className={parentStyle.date}>
                             <span className={style.columnLabel}>
-                                Date:
+                                {t('transaction.details.date')}:
                             </span>
                             <span className={style.date}>{sDate}</span>
                         </div>
-                        <div className={parentStyle.address}>
-                            <span className={style.columnLabel}>
-                                Address:
-                            </span>
-                            <span className={style.address}>
-                                {addresses[0]}
-                                {
-                                    addresses.length > 1 && (
+                        { note ? (
+                            <div className={parentStyle.activity}>
+                                <span className={style.address}>
+                                    {note}
+                                </span>
+                            </div>
+                        ) : (
+                            <div className={parentStyle.activity}>
+                                <span className={style.label}>
+                                    {t(`transaction.tx.${type === 'receive' ? 'received' : 'sent'}`)}
```

Comment:
it is better to keep the full t(...) string always explicit, so it is easier to look for them later, or to automatically find unused ones etc. so better to do:

`type === 'receive' ? t('transaction.tx.received') : t('transaction.tx.sent')`.

Reply:
You are right searching for `transaction.tx.received` is nice. I'll change it to:

`t(type === 'receive' ? 'transaction.tx.received' : 'transaction.tx.sent')`